tests/system/aiplatform/test_model_upload.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import tempfile
+import importlib
+
+import pytest
+
+from google import auth as google_auth
+from google.cloud import aiplatform
+from google.cloud import storage
+from google.cloud.aiplatform import initializer
+
+from tests.system.aiplatform import e2e_base
+
+# TODO(vinnys): Replace with env var `BUILD_SPECIFIC_GCP_PROJECT` once supported
+_, _TEST_PROJECT = google_auth.default()
+_TEST_LOCATION = "us-central1"
+
+_XGBOOST_MODEL_URI = "gs://ucaip-test-us-central1/models/iris_xgboost/model.bst"
+
+
+class TestModel(e2e_base.TestEndToEnd):
+    @pytest.mark.usefixtures("teardown")
+    def test_upload_and_deploy_xgboost_model(self, shared_state):
```

Comment:
```suggestion

@pytest.mark.usefixtures("delete_staging_bucket", "teardown")
class TestModel(e2e_base.TestEndToEnd):
    _temp_prefix = 'local-model-upload'
    
    def test_upload_and_deploy_xgboost_model(self, shared_state):
```

Reply:
`_temp_prefix` is required to be defined

`'delete_staging_bucket'` will delete the created staging bucket and usage in the test is correct

Moved the fixtures to class level in case we want to add additional tests.

Reply:
Done. Thank you for the example.

P.S. I wonder, what is the main motivation to use the `teardown` fixture as opposed to the `teardown_class`/`teardown_method` methods?